### PR TITLE
Fix Collage Block layouts for the TwentyTwenty theme

### LINKED ISSUE #1062: ISBAT use Collage block in TwentyTwenty theme
**Describe the bug**
Running TwentyTwenty theme the Collage block when published does not reflect the editor view. 

**To Reproduce**

1. Add Collage block. 
2. Populate images.
3. Publish page and view page as a user.
4. See error.

**Screenshots**
**Collage block in editor**
image

**Collage block on pushed page**
image

**Isolating the problem:**
Mark completed items with an [x].

-   [x] This bug happens with no other plugins activated
-   [x] This bug happens with a default WordPress theme active
-   [x] This bug happens **without** the Gutenberg plugin active
-   [x] I can reproduce this bug consistently using the steps above

**WordPress Version**
What version of WordPress are you using?
5.3 RC2 

**Gutenberg Version**
What version of the Gutenberg plugin are you using? (If any)
N/A

### PR DESCRIPTION
**Pull request by paranoia1906**
Closes #1062.

Pull request add `margin: inherit` to `li` elements scoped for collage block. This change allows correct display in TwentyTwenty. This change still looks correct in TwentyNineteen, TwentySeventeen and Barebones theme (underscores).

### CODE REVIEW

**Comment by paranoia1906** [comment]
Block with Fix in place.
image

File: src/blocks/gallery-collage/styles/style.scss [hunk 1]
```diff
@@ -12,6 +12,10 @@
 		list-style-type: none;
 		margin: 0;
 		padding: 0;
+
+		li {
+			margin: inherit;
+		}
```
Comment (paranoia1906) [inline src/blocks/gallery-collage/styles/style.scss]: In terms of targeting only `.is-twentytwenty`, I am not sure that we should limit in this case. `table-layout: auto` seemed to render correctly in the different themes. What are your thoughts?

**Comment by richtabor** [comment]
Editor instance also needs improvements: Screen Shot 2019-10-24 at 8 47 49 PM

**Comment by richtabor** [comment]
_Fixed now._ 

Editor: 
Screen Shot 2019-10-24 at 8 52 21 PM

Frontend: 
Screen Shot 2019-10-24 at 8 53 08 PM

**Pull request merged**